languages/common-lisp/concepts/floating-point-numbers/about.md

```diff
@@ -1 +1,31 @@
-TODO: add information on floating-point-numbers concept
+Floating-Point Numbers are fractional or whole numbers including a decimal point (like `3.14`, `-1.7`, `99.99`, `2048.0`)
```

Comment:
This document is probably where we want to get into discussion of single- and double-float (oh and it looks like the spec also specifcies short- and long- floats...)